### ISSUE TITLE
Add ARIA support to Mobile Menu/Offcanvas toggle for improved accessibility

### ISSUE THREAD

**Issue by N8Solutions** [gantry/gantry5#1891]
In reviewing Gantry 5 for accessibility enhancements I noticed the Mobile Menu / Offcanvas Toggle has no ARIA attributes associated with it.

I do not believe `aria-hidden` should be added to the icon since it is an Interactive Element nor should it only be recognized as a menu because it can also be set to be visible on tablets and larger screens that may not contain menu items.

I would suggest the ARIA support be added in a way so that it recognizes the Toggle as the **Menu** when the screen size is that of a mobile and something like **Expand Offcanvas**.

I found this issue on GitHub from Automattic and how they addressed the issue of accessibility with the Toggle.
https://github.com/Automattic/_s/issues/545

**Comment by mahagr** [comment]
https://github.com/Automattic/_s/commit/e142af3a2720e38179e294181562c22842a68c3e

Great idea. I'm assigning this to @w00fz as there's javascript change which needs to be done.

**Comment by mahagr** [comment]
By default the particle div should have: `aria-controls="menu" aria-expanded="false"`. When mobile menu gets shown, javascript should change `aria-expanded="true"` and back to `false` depending on the state of the menu.

**Comment by N8Solutions** [comment]
@w00fz You're awesome Djamil!  Thank you for taking care of this!
